Dockerfile

```diff
@@ -1,9 +1,33 @@
+# FROM python:latest
+# WORKDIR /usr/src/app
+# COPY . .
+
+# EXPOSE 8000
+
+# RUN apt-get update && apt-get install -y \
+#     build-essential \
+#     python3-pip \
+#     && pip3 install mkdocs
+
+
+# RUN make install-python-requirements
+# RUN make generate-site
+# ENTRYPOINT ["make", "serve"]
+
+##
 FROM python:latest
 WORKDIR /usr/src/app
 COPY . .
 
 EXPOSE 8000
 
+RUN apt-get update && apt-get install -y \
+    build-essential \
+    python3-pip \
+    dos2unix \
+    && pip3 install mkdocs
+
+RUN dos2unix scripts/Generate_Site_mkDocs.sh
```

Comment:
Okay. I do not like this. If someone using Windows previously edited a bash script so it ended up with \r\n as EOL terminators rather than just \n and broke the script, the better way to deal with this is to create an issue for the particular script and just fix the script. This is a kludge, and if we start using this, we may eventually find ourselves having to do this for all the bash scripts under 'scripts'.

Furthermore, if this is becoming a problem, a better way to address it is to have contributors set the 'core.input' attribute in their .gitconfig file. In ESAPI, we mention this:
> Finally, we recommend setting the git property 'core.autocrlf' to 'input' in your $HOME/.gitconfig file; e.g., that file should contain something like this:
> 
> [core]
> &nbsp;&nbsp;&nbsp;&nbsp;autocrlf = input

and since people have been using that, we haven't had that problem. (And, if they ignore that and persist on messing up bash scripts by using \r\n for EOL terminators, we force them to program for 3 months while wearing mittens.)

So, of course, if we don't execute dos2unix, we don't need to install it either line 27 so it can be removed from there as well.